screens/GameScreen.java

```diff
@@ -172,7 +183,10 @@ public void paintComponent(Graphics g) {
 			target.draw(g);
 		}
 		player.draw(g);
-		for(DamageMarker marker : damageMarkers) {
+		for (Animation animation : animations) {
```

Comment:
I still think we can keep the number of arraylists to a minimum, but let's add a todo for refactoring here for now. Once animations are done, we can see how to refactor.